### PR TITLE
created employee education history model

### LINKED ISSUE #11: Create employee education history model
Create an employee education history model

attributes to include can be referred in the docs link: <ink to be added>

added model with appropriate key values and conditions.
Models should be added in the model folder under the file and model name of `emp_edu_history`

Also, make object of values that share the same parent
for eg:
```
GRADUATION BRANCH    
GRADUATION DEGREE    
GRADUATION MARK PCT    
GRADUATION PASSING YEAR 
```
can be put as:
```javascript
{
graduation: { 
    institute: Value, 
    branch: Value, 
    ...}
}```

### PR DESCRIPTION
**Pull request by kskarthik5**
fixes #11

### CODE REVIEW

File: models/emp_edu_history.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    from_year: { type: String, required: true },
+    upto_year: { type: String, required: true },
+    registration_number: { type: String, required: true },
+    aggregate_pct: { type: String, required: true },
+    final_year_pct: { type: String, required: true },
+    number_of_attempts: { type: Number, required: true },
+    rank: { type: Number, required: true },
+    passing_year: { type: String, required: true },
+}
+const employmentEducationHistorySchema = {
```
Comment (Hitansh159) [inline models/emp_edu_history.js:19]: rename variable to employeEducationHistoryScehema

File: models/emp_edu_history.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ssc: { type: Education, required: true },
+    hsc: { type: Education, required: true },
+    dip: { type: Education },
+    iti: { type: Education },
+    deg: { type: Education },
+    pgd: { type: Education },
+    phd: { type: Education },
+    pdoc: { type: Education }
+}
+const EmploymentEducationHistory = new connector.model('emp-edu-history',employmentEducationHistorySchema);
```
Comment (Hitansh159) [inline models/emp_edu_history.js:29]: rename variable to employeEducationHistory

**Review by Hitansh159 — CHANGES_REQUESTED** [review]
Overall looks great just change variable names and its good to go

**Comment by kskarthik5** [comment]
ah mb. is this fine now?

**Review by Hitansh159 — APPROVED** [review]

**Pull request merged**